packages/batch-submitter/package.json

```diff
@@ -36,8 +36,8 @@
     "@eth-optimism/contracts": "^0.4.5",
     "@eth-optimism/core-utils": "^0.5.1",
     "@eth-optimism/ynatm": "^0.2.2",
-    "@ethersproject/abstract-provider": "^5.0.5",
-    "@ethersproject/providers": "^5.0.14",
+    "@ethersproject/abstract-provider": "^5.4.0",
+    "@ethersproject/providers": "^5.4.0",
```

Comment:
Should probably bump this version to 5.4.3 to accommodate [fix to forward missing EIP-1559 fields to call and estimateGas](https://github.com/ethers-io/ethers.js/releases/tag/v5.4.3)